### ISSUE TITLE
Why sigmoid on es and eso?

### ISSUE THREAD

**Issue by SuminHan** [LMissher/ST-GRAT#3]
model.py at line 155 and 165:

        es = torch.sigmoid((query[:self.k//2] * keys[:self.k//2]).sum(-1, keepdim=True) / (self.d ** 0.5))
        eso = torch.sigmoid((query[self.k//2:] * keys[self.k//2:]).sum(-1, keepdim=True) / (self.d ** 0.5))

Unlike the paper, I wonder why you put sigmoid on here.

**Comment by LMissher** [comment]
I use sigmoid to keep the positive of es and eso. If es and eso are not positive, the es + torch.exp(e).sum(-1, keepdim=True) maybe near to zero and lead to fp32 overflow.
